conda-forge.yml

```
@@ -18,5 +22,6 @@ github:
   tooling_branch_name: main
 provider:
   linux_aarch64: default
+  linux_ppc64le: default
   win: azure
 test: native_and_emulated
```

Testing isn't working in the cross-compiled case, which is as expected. So this will limit testing to native only

```suggestion
test: native
```